front-end/src/screens/SignupForm/index.tsx

```diff
@@ -2,208 +2,32 @@
-								/>
-							</label>
-							<input
-								className={errors.name ? 'error' : ''}
-								name='name'
-								placeholder='Firstname Lastname'
-								type='text'
-								ref={register({ maxLength: 30, minLength: 3, required: false })}
-							/>
-							{(errors.name as FieldError)?.type === 'maxLength' && <span className={'errorText'}>{messages.VALIDATION_NAME_MAXLENGTH_ERROR}</span>}
-							{(errors.name as FieldError)?.type === 'minLength' && <span className={'errorText'}>{messages.VALIDATION_NAME_MINLENGTH_ERROR}</span>}
-						</Form.Field>
-					</Form.Group>
-					<Form.Group>
-						<Form.Field width={16}>
-							<label>
-								Email
-								<HelperTooltip
-									content='Your email is used for password recovery or discussion notifications.'
-								/>
-							</label>
-							<input
-								className={errors.email ? 'error' : ''}
-								name='email'
-								placeholder='john@doe.com'
-								ref={register({
-									pattern: /^[A-Z0-9_'%=+!`#~$*?^{}&|-]+([.][A-Z0-9_'%=+!`#~$*?^{}&|-]+)*@[A-Z0-9-]+(\.[A-Z0-9-]+)+$/i
-								})}
-								type='text'
-							/>
-							{errors.email && <span className={'errorText'}>{messages.VALIDATION_EMAIL_ERROR}</span>}
-						</Form.Field>
-					</Form.Group>
-					<Form.Group>
-						<Form.Field width={16}>
-							<label>Password<sup>*</sup></label>
-							<input
-								className={errors.password ? 'error' : ''}
-								name='password'
-								placeholder='Password'
-								ref={register({ minLength: 6, required: true })}
-								type='password'
-							/>
-							{errors.password && <span className={'errorText'}>{messages.VALIDATION_PASSWORD_ERROR}</span>}
-						</Form.Field>
-					</Form.Group >
-					<Form.Field>
-						<label className='checkbox-label'>
-							<input
-								className={errors.termsandconditions ? 'error' : ''}
-								name='termsandconditions'
-								value='yes'
-								ref={register({ required: true })}
-								type='checkbox'
-							/>
-							I have read and agree to the terms of the <Link to='/terms-and-conditions'>Polkassembly end user agreement</Link>.
-						</label>
-						{errors.termsandconditions && <div className={'errorText'}>Please agree to the terms of the Polkassembly end user agreement.</div>}
-					</Form.Field>
-					<div>To see how we use your personal data please see our <Link to='/privacy'>privacy notice</Link>.</div>
-					<div className={'mainButtonContainer'}>
-						<Button
-							primary
-							disabled={loading}
-							onClick={handleSubmitForm}
-							type="submit"
-						>
-							Sign-up
-						</Button>
-						{error?.message && <FilteredError text={error.message}/>}
-					</div>
-				</Form>
+				{ displayWeb2
+					? <Web2Signup toggleWeb2Signup={toggleWeb2Signup}/>
+					: <Web3Signup toggleWeb2Signup={toggleWeb2Signup}/>
+				}
 			</Grid.Column>
+			<Grid.Column only='tablet computer' tablet={2} computer={4} largeScreen={5} widescreen={5}/>
```

Comment:
I think that one isn't needed. Also if it was, it should be identical to the first one.
```suggestion
```